go.sum

```diff
@@ -98,6 +98,8 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
 github.com/DataDog/agent-payload v4.80.0+incompatible h1:fYmbV/oW3rQub6add/Ikldh8t/SyI47oVJsHg1PhqVw=
 github.com/DataDog/agent-payload v4.80.0+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.81.0+incompatible h1:CCALFJ0VC5rO0PW9nLVh1fGPmBr7Ky4xHhk4WMJZJDI=
+github.com/DataDog/agent-payload v4.81.0+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
```

Comment:
I think you need to run `go mod tidy` to not keep adding the versions.

Reply:
Oh thanks! I was wondering what the best way to do that was